analyticConfigurations/computed_metric-aws.rds.json

```diff
@@ -4,55 +4,72 @@
         "match": "netuitive.aws.rds.averagereadsize",
         "properties": {
-          "expression": "data['aws.rds.readiops'].actual == 0 ? 0 : data['aws.rds.readthroughput'].actual / data['aws.rds.readiops'].actual",
+          "expressions": [
+            "${aws.rds.readthroughput}.actual / ${aws.rds.readiops}.actual",
+            "0"
+          ],
           "fqn": "netuitive.aws.rds.averagereadsize",
           "name": "Average Read Size"
         }
       },
       {
         "match": "netuitive.aws.rds.averagewritesize",
         "properties": {
-          "expression": "data['aws.rds.writeiops'].actual == 0 ? 0 : data['aws.rds.writethroughput'].actual / data['aws.rds.writeiops'].actual",
+          "expressions": [
+            "${aws.rds.writethroughput}.actual / ${aws.rds.writeiops}.actual",
+            "0"
+          ],
           "fqn": "netuitive.aws.rds.averagewritesize",
           "name": "Average Write Size"
         }
       },
       {
         "match": "netuitive.aws.rds.connectionthroughpututilizationpercent",
         "properties": {
-          "expression": "data['aws.rds.databaseconnections'].actual == 0 ? 0 : 100 * (data['aws.rds.readlatency'].actual * data['aws.rds.readiops'].actual + data['aws.rds.writelatency'].actual * data['aws.rds.writeiops'].actual) / data['aws.rds.databaseconnections'].actual",
+          "expressions": [
+            "100 * (${aws.rds.readlatency}.actual * ${aws.rds.readiops}.actual + ${aws.rds.writelatency}.actual * ${aws.rds.writeiops}.actual) / ${aws.rds.databaseconnections}.actual",
+            "0"
+          ],
           "fqn": "netuitive.aws.rds.connectionthroughpututilizationpercent",
           "name": "Connection Throughput Utilization Percent"
         }
       },
       {
         "match": "netuitive.aws.rds.diskspacepercentused",
         "properties": {
-          "expression": "((attribute['allocatedStorage'].value - (data['aws.rds.freestoragespace'].actual/1024/1024/1024)) / attribute['allocatedStorage'].value) * 100",
+          "expressions": [
+            "100 * (attribute(allocatedStorage) - (${aws.rds.freestoragespace}.actual/1024/1024/1024)) / attribute(allocatedStorage)"
+          ],
           "fqn": "netuitive.aws.rds.diskspacepercentused",
           "name": "Disk Space Percent Used"
         }
       },
       {
         "match": "netuitive.aws.rds.iopsutilization",
         "properties": {
-          "expression": "Math.min(100, data['netuitive.aws.rds.totaliops'].actual * 100 / (!attribute['iops'].value ? (attribute['allocatedStorage'].value * 3 < 100 ? 100 : (attribute['allocatedStorage'].value * 3 > 10000 ? 10000 : attribute['allocatedStorage'].value * 3)) : attribute['iops'].value))",
+          "expressions": [
+            "Math.min(100, ${netuitive.aws.rds.totaliops}.actual * 100 / (!attribute(iops) ? (attribute(allocatedStorage) * 3 < 100 ? 100 : (attribute(allocatedStorage) * 3 > 10000 ? 10000 : attribute(allocatedStorage) * 3)) : attribute(iops)))"
```

Comment:
@jemassey Does `Math.min` still exist?

Reply:
Apparently not because this metric doesn't work anymore. @jemassey what's the best method to use here?

Reply:
I cannot think of a good workaround...can not convert this one yet...